src/test/java/org/cactoos/scalar/MaxOfTest.java

```diff
@@ -39,114 +38,193 @@
 
     @Test
     public void withIntegerCollection() {
-        MatcherAssert.assertThat(
-            new MaxOf(
-                new ListOf<>(1, 2, 3, 4).toArray(new Integer[4])
-            ).intValue(),
-            Matchers.equalTo(4)
-        );
-        MatcherAssert.assertThat(
-            new MaxOf(
-                new ListOf<>(1, 2, 3, 4).toArray(new Integer[4])
-            ).longValue(),
-            Matchers.equalTo(4L)
-        );
-        MatcherAssert.assertThat(
-            new MaxOf(
-                new ListOf<>(1, 2, 3, 4).toArray(new Integer[4])
-            ).doubleValue(),
-            Matchers.equalTo(4.0d)
-        );
-        MatcherAssert.assertThat(
-            new MaxOf(
-                new ListOf<>(1, 2, 3, 4).toArray(new Integer[4])
-            ).floatValue(),
-            Matchers.equalTo(4.0f)
-        );
+        new Assertion<>(
+            "must maximum integer of positive integers",
```

Comment:
@fabriciofx the assertion messages need to be fixed. Eg. this one should read "must select maximum integer of positive integers"

Reply:
@llorllale done.